app/views/admin/products/new.html.erb

```diff
@@ -1,3 +1,21 @@
 <h1>New Product</h1>
 
-<%= render partial: 'product_form' %>
```

Comment:
Thanks! I know the original way was to render partial but I had problems because I needed the partial forms to route differently for new products and edited products.